tests/Fixtures/Integration/set/@PER-CS2.0_php80.test-in.php

```diff
@@ -4,6 +4,9 @@ function parametersMultilineWithoutComma(
     $x,
     $y
 ) {}
+
+parametersMultilineWithoutComma(x   :   3, y   : 4);
```

Comment:
Since the spaces after the colon are also changed the name of the fixer is misleading.